include/mbedtls/ecdsa.h

```diff
@@ -212,11 +213,6 @@ int mbedtls_ecdsa_write_signature( mbedtls_ecdsa_context *ctx, mbedtls_md_type_t
  *          serialized as defined in <em>RFC-4492: Elliptic Curve Cryptography 
  *          (ECC) Cipher Suites for Transport Layer Security (TLS)</em>.
  *
- *          The deterministic version, defined in <em>RFC-6979: 
```

Comment:
Suggestion: Don't remove the comment as a whole, but only the conditional `if #MBEDTLS_ECDSA_DETERMINISTIC is defined`. I think it's good to always make clear if the deterministic version is used or not, or what its use depends on.

Reply:
accepted